mysensors/handler.py

```diff
@@ -135,6 +135,7 @@ def handle_stream(msg):
     if not msg.gateway.is_sensor(msg.node_id):
         return None
     stream = msg.gateway.const.Stream(msg.sub_type)
+    msg.gateway.alert(msg)
```

Comment:
I'm wondering if we should alert after the final stream message handler is called. That is done below on the same line we return. We could save the return value of the message handler in a variable, alert and then return the return value.

This is the pattern we use for all other messages, that we alert after the message handler has run. What do you think?